pkg/pods/pod.go

```diff
@@ -229,11 +229,12 @@ func (pod *Pod) buildRunitServices(launchables []launch.Launchable, restartPolic
 				return util.Errorf("Duplicate executable %q for launchable %q", executable.Service.Name, launchable.ID())
 			}
 			sbTemplate[executable.Service.Name] = runit.ServiceTemplate{
+				Log: newManifest.GetLogExecutable(),
```

Comment:
I agree with you - we need to think through the configuration a bit more. Is this code path the right place to express a default log exec? For gradual rollout and general configurability I want the pod manifest to override any default that P2 supplies.

Fully with you that the kafka logbridge is the right thing for 99% of cases and it will eventually become the default.